cypress/e2e/Pages/LoginPage.js

```diff
@@ -0,0 +1,32 @@
+class loginPage {
+    visit() {
+      cy.visit('https://www.saucedemo.com/');
+    }
+  
+    setUsername() {
+      cy.get('#user-name').clear().type('standard_user');
+    }
+  
+    setPassword() {
+      cy.get('#password').type('secret_sauce');
```

Comment:
Passwords better to externalize with using dotEnv package
https://www.npmjs.com/package/dotenv

Reply:
thanks, I will correct that